openpifpaf/show/annotation_painter.py

```diff
@@ -0,0 +1,35 @@
+from collections import defaultdict
+
+from .painters import KeypointPainter, CrowdPainter, DetectionPainter
+
+PAINTERS = {
+    'Annotation': KeypointPainter,
+    'AnnotationCrowd': CrowdPainter,
+    'AnnotationDet': DetectionPainter,
+}
+
+class AnnotationPainter:
+    def __init__(self, *,
+                 xy_scale=1.0,
+                 painters=None):
+        self.painters = {}
+        for annotation_type, painter in PAINTERS.items():
```

Comment:
When the dictionary of `painters` has more entries than `PAINTERS`, it would ignore those.
Instead, can you initialize `self.painters` with a dict-comprehension from `PAINTERS` and then add or overwrite with entries from `painters`?